### DISCUSSION TITLE
alt text for images

### DISCUSSION THREAD

**Discussion by sl33p** [microfeed/microfeed#27] [Ideas] [answered]
Hi, thanks for the great project, and especially for the fantastic documentation.

Is there a plan to support adding alt text to images on the roadmap, and/or is there a way to hack it currently?

Thanks,

Nicklas

**Comment by wenbinf** [comment, marked as answer]
Yes, you can edit raw html/css at /admin/settings/code-editor/?type=themes&theme=custom#webFeed

<img width="639" alt="Screenshot 2023-01-03 at 8 46 26 AM" src="https://user-images.githubusercontent.com/1719237/210285409-2066bc84-948e-4f12-a0f2-dc19d28410b7.png">

**Reply by sl33p** [reply]
Thanks, but if I wanted to add an alt text for each image I would need to store it with the item. Unless I wanted to reuse an existing field that's not possible  as far as I understand it?

**Reply by wenbinf** [reply]
I see.. So you want to add alt for embedded images that are uploaded right from the Rich Editor.

For now, there's not an easy way to do so, but you can edit raw html:

<img width="1019" alt="Screenshot 2023-01-03 at 9 39 28 AM" src="https://user-images.githubusercontent.com/1719237/210287497-41a8cbc5-ea14-4d29-99b9-febfbed139cb.png">

TODO for us:
* Make a better rich editor :)

**Comment by amblerkr** [comment]
That's a good suggestion. Improved accessibility!
